lib/amqp_client.js

```diff
@@ -169,11 +169,11 @@ AMQPClient.prototype.connect = function(url) {
 };
 
 /**
- * Creates a sender link for the given address, with optional link options
+ * Creates a sender link for the given address, with optional link policy
  *
  * @method createSender
- * @param {string} [address]    An address to connect this link to
- * @param {*} [options]         Options used for creating the link
+ * @param {String} [address]    An address to connect this link to
+ * @param {Object} [options]    Policy used for creating the sender link
```

Comment:
Well, what it _really_ is are `linkPolicyOverrides` or `senderPolicyOverrides`, but if that's too long I'd understand. If so, `senderPolicy` is my preference.